beamer/l1_resolution.py

```diff
@@ -1,12 +1,29 @@
+import shutil
 import subprocess
 
+import structlog
 from hexbytes import HexBytes
 
 from beamer.typing import URL
 
+
+log = structlog.get_logger(__name__)
+
 RELAYER_EXECUTABLE = "beamer-relayer"
 
 
+def healthcheck() -> bool:
+    """Checks if the relayer executable is in PATH, return `False` if not"""
```

Comment:
This is a weird name for a function that checks the existence of a executable.

Reply:
Any recommendation?

Reply:
`relayer_executable_exists` maybe? Something along those lines, if we're going to retain the current `which` approach.